imagedephi/main.py

```diff
@@ -24,12 +27,31 @@ def imagedephi() -> None:
     "input-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
 )
 @click.argument(
-    "output-dir", type=click.Path(exists=True, file_okay=False, writable=True, path_type=Path)
+    "output-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
```

Comment:
If it needs to be both readable and writable, I'd suggest explicitly adding both kwargs.

Reply:
a66dc07 ensures that the output dir is writable. I think we want both, right? We want to know what files already exist, in case we need to overwrite, and we need to be able to write the redacted images to the directory.